app/src/main/java/com/myvetpath/myvetpath/User.java

```diff
@@ -0,0 +1,68 @@
+package com.myvetpath.myvetpath;
+
+import java.sql.Date;
+
+public class User { //remaned to picture so we don't get it confused with Image object from android
+    //Table Fields
+    public static final String TABLE_NAME = "User";
+    public static final String COLUMN_ID = "UserID";
+    public static final String COLUMN_USERNAME = "UserName";
+    public static final String COLUMN_PASSWORD = "Password";
+    public static final String COLUMN_AUTHORIZED = "Authorized";
+
+    //Create Table String
+    public static final String CREATE_TABLE =
+            "CREATE TABLE " + TABLE_NAME + "(" + COLUMN_ID +
+                    "INTEGER PRIMARY KEY,"
+                    + COLUMN_USERNAME + "TEXT"
+                    + COLUMN_PASSWORD + "TEXT"
+                    + COLUMN_AUTHORIZED + "INTEGER" + ")";
+
+
+    //Object Fields
+    private int userID;
+    private String userName;
+    private String password;
+    private int authorized; //flag variable 0 means no 1 means yes.
```

Comment:
Can this only be 0 or 1? If so, maybe it would be easier to use a boolean.

Reply:
Yes I just wasn't sure what the boolean type was in sqlite